src/EA.Weee.Domain.Tests.Unit/AatfReturn/ReturnSchemeTests.cs

```diff
@@ -0,0 +1,37 @@
+﻿namespace EA.Weee.Domain.Tests.Unit.AatfReturn
+{
+    using EA.Weee.Domain.AatfReturn;
+    using FakeItEasy;
+    using FluentAssertions;
+    using System;
+    using System.Collections.Generic;
+    using System.Linq;
+    using System.Text;
+    using System.Threading.Tasks;
+    using Xunit;
+
+    public class ReturnSchemeTests
+    {
+        [Fact]
+        public void ReturnScheme_GivenSchemeIsNull_ThrowsArgumentNullException()
+        {
+            Action constructor = () =>
+            {
+                var @return = new ReturnScheme(null, A.Fake<Return>());
+            };
+
+            constructor.Should().Throw<ArgumentNullException>();
+        }
```

Comment:
Add unit test to check that the properties of the class are set in the  constructir